models/quality_measures/quality_measures_models.yml

```diff
@@ -131,6 +131,57 @@ models:
       - name: tuva_last_run
         description: The date and timestamp of the dbt run.
 
+## Shared Exclusions
+  - name: shared_exclusions__frailty
+    config:
+      schema: |
+        {%- if var('tuva_schema_prefix',None) != None -%}{{var('tuva_schema_prefix')}}_shared_exclusions{% else %}shared_exclusions{%- endif -%}
```

Comment:
When we do the PR to the main repo, let's change the schema config back to quality_measures/_quality_measures to ensure all tables in this mart get created under the same schema/prefix.